backend/app/modules/scheduler/SchedulerActor.scala

```diff
@@ -48,18 +50,29 @@ class SchedulerActor(processes: Queries, configuration: play.api.Configuration,
     * * Download image
     */
   def startProcessing(id: String): Unit = {
-      monitoring.ask(GetProcess(id))
+      val details = monitoring
```

Comment:
`details` is a bit obscure here. What is it? Process details? if so, use `process_details`. Or is it download details ? if so, use `download_details`.